bin/run_app.py

```diff
@@ -1,38 +1,63 @@
 __author__ = 'mwham'
-import sys
 import os
-import multiprocessing
+import argparse
 import logging
 import logging.config
 import logging.handlers
-sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-import reporting_app
-import rest_api
-from config import rest_config, reporting_app_config
+
+watched_files = [
+    os.path.join(os.path.dirname(__file__), '..', 'etc', 'schema.yaml')
```

Comment:
Possibly a badly-chosen variable name there - Werkzeug and Tornado watch the files for all loaded modules as standard, and can be configured to watch additional files (in this case, anything listed in `watched_files`)